lib/msal-angular/package.json

```diff
@@ -74,7 +74,7 @@
     "ts-loader": "^2.0.3",
     "tslib": "^1.7.1",
     "tslint": "^5.8.0",
-    "typedoc": "^0.6.0",
+    "typedoc": "^0.14.2",
```

Comment:
maybe pull dev into your integration branch so this doesn't show up

Reply:
I'm not sure why this is showing up. I have merged the changes from dev into the integration branch...